terraform/load_balancer.tf

```diff
@@ -0,0 +1,30 @@
+resource "digitalocean_loadbalancer" "lb" {
+  name                     = "lb-1"
+  region                   = "ams3"
+  enable_backend_keepalive = true
+  droplet_tag              = "web-lb"
+
+  forwarding_rule {
+    entry_port      = 80
+    entry_protocol  = "http"
+
+    target_port     = 80
+    target_protocol = "http"
+  }
+
+  sticky_sessions {
+    type               = "cookies"
+    cookie_name        = "DO-LB"
+    cookie_ttl_seconds = 300
+  }
+
+  healthcheck {
+    protocol                 = "http"
+    port                     = 80
+    path                     = "/"
```

Comment:
I've added a health check endpoint in #152, but maybe we can just change this later to use that.

Reply:
I see, maybe we can just add it once it's merged.